packages/thumbprint-react/components/Calendar/index.module.scss

```diff
@@ -113,30 +99,37 @@
     height: $cell-side;
     width: $cell-side;
     min-width: 44px;
-    padding-top: $vertical-padding;
-    padding-bottom: $vertical-padding;
 }
 
-.root :global(.DayPicker-Day--disabled) {
-    color: $tp-color__gray;
-    cursor: default;
+.root :global(.rdp-cell:has(div)) {
```

Comment:
`:has` doesn't work in Firefox yet. Is there a different way of doing this? https://caniuse.com/css-has

Reply:
I tried several options, but couldn't find anything relevant. Also posted on recat-day-picker [issues](https://github.com/gpbl/react-day-picker/issues/1854).

Reply:
Oh, nice work! Sounds like they're going to fix it soon. Can you QA and see how looks in Firefox? Do we need to wait until they fix it, or is it usable?

Reply:
I tested it in Firefox and it's look like this. Author has added it to milestone for react-day-picker@8.9.0.  So, not sure when it will get fixed. I tested functionality, everything looks good to me except minor cell border for outside day cells. @danoc What you suggest?  Should we can go with it or should we wait for fix in firefox? (Also testing in Safari & Chrome where it's LGTM.)

<img width="371" alt="Screenshot 2023-08-02 at 5 12 28 PM" src="https://github.com/thumbtack/thumbprint/assets/104916135/669a56a9-0c49-4dfa-bb56-206eb7732925">

Reply:
@danoc  Thoughts?

Reply:
Taking it as a non-blocker and going to merge CR.